src/fprime_gds/common/decoders/file_decoder.py

```diff
@@ -57,8 +57,4 @@ def decode_api(self, data):
         if packetType == "END":  # Packet Type is END
             hashValue = struct.unpack_from(">I", data, 5)
             return [file_data.EndPacketData(seqID, hashValue),]
-        if packetType == "CANCEL":  # Packet Type is CANCEL
-            # CANCEL Packets have no data
-            return [file_data.CancelPacketData(seqID),]
-        # The data was not determined to be any of the packet types so return none
-        return None
+        return [file_data.CancelPacketData(seqID),] if packetType == "CANCEL" else None
```

Comment:
Same as above. If statements are nice but I (personally) don't think they should be used everywhere they can be

Reply:
Done.